content/3_Silver/Two_Pointers.mdx

```diff
@@ -44,16 +44,146 @@ of an interval. It can also be used to track two values in an array as shown in
 
 We want to find the longest contiguous segment of books that can be read within $t$ minutes. 
 
-To accomplish this, we can define $\texttt{left}$ and $\texttt{right}$ as the start and end of the segment respectively and use two pointers to track the interval.
+To accomplish this, we can define $\texttt{left}$ and $\texttt{right}$ both at the start of the segment respectively and use two pointers to track the interval.
```

Comment:
i think the former is correct 👀

Reply:
this is for books, right? i'm pretty sure both pointers start at the beginning

but delete "respectively"

Reply:
but right marks the end of the segment not the start even though they all start at the same point at the beginning.

Reply:
ohhh i thought it was saying that they both start at the beginning, and the "tracks the interval" part meant that they were the left and right side of the segment

maybe could be reworded to make it more clear

Reply:
yea i think we can reword this to be a bit clearer

Reply:
Yeah let me reword that